localization/react-intl/src/app/components/source/UserPrivacy.json

```diff
@@ -39,6 +39,10 @@
     "id": "userPrivacy.stopProcessingButton",
     "defaultMessage": "Request to stop processing"
   },
+  {
+    "id": "userPrivacy.connected_accounts",
```

Comment:
sorry to bother with this, but we are usually using `CamelCase` instead of `snake_case` for those ids :)